tools/json-to-module/main.js

```diff
@@ -262,7 +262,7 @@ async function assembleManifest(args) {
                 twitter: "@ForgeVTT",
             },
         ],
-        url: `https://www.dndbeyond.com/sources/${args.book}`,
+        url: `https://www.dndbeyond.com/sources/dnd/${args.book}`,
```

Comment:
Here is the simple fix that had me overwriting the module files and bring the update load.